cmd/create/cluster/cmd.go

```diff
@@ -2923,6 +3066,27 @@ func buildCommand(spec ocm.Spec, operatorRolesPrefix string,
 		command += fmt.Sprintf(" --audit-log-arn %s", *spec.AuditLogRoleARN)
 	}
 
+	if !reflect.DeepEqual(spec.DefaultIngress, ocm.DefaultIngressSpec{}) {
+		if len(spec.DefaultIngress.RouteSelectors) != 0 {
+			command += fmt.Sprintf(" --%s \"", defaultIngressRouteSelectorFlag)
+			for k, v := range spec.DefaultIngress.RouteSelectors {
+				command += fmt.Sprintf("%s=%s", k, v)
+			}
+			command += "\""
+		}
+		if len(spec.DefaultIngress.ExcludedNamespaces) != 0 {
+			command += fmt.Sprintf(" --%s %s", defaultIngressExcludedNamespacesFlag,
+				strings.Join(spec.DefaultIngress.ExcludedNamespaces, ","))
+		}
+		if spec.DefaultIngress.WildcardPolicy != "" {
+			command += fmt.Sprintf(" --%s %s", defaultIngressWildcardPolicyFlag, spec.DefaultIngress.WildcardPolicy)
+		}
+		if spec.DefaultIngress.NamespaceOwnershipPolicy != "" {
+			command += fmt.Sprintf(" --%s %s", defaultIngressNamespaceOwnershipPolicyFlag,
+				spec.DefaultIngress.NamespaceOwnershipPolicy)
+		}
```

Comment:
If none was selected, the built command has this flag without an argument, which causes the command to fail. 
```
Failed to execute root command: flag needs an argument: --default-ingress-namespace-ownership-policy
```
This is likewise the case with the other option that has a 'none' option as well.

Reply:
Thanks for catching it, I added the printing commands at a later moment